src/applications/disability-benefits/all-claims/tests/all-claims-keyboard-only.cypress.spec.js

```diff
@@ -1,430 +1,432 @@
-
-    // For some reason using `cy.tabToContinueFormSimulatedEnter` here doesn't work. It
-    // instead will skip ahead to the following page.
-    cy.tabToElementAndPressSpace('button[type="submit"]');
-
-    // I. Veteran Information > I. Separation Pay
-    // ==========================================
-    cy.url().should('include', veteranDetailsPages.separationPay.path);
-    cy.injectAxeThenAxeCheck();
-
-    // 1. Can provide separation pay details if received
-    cy.tabToElement('[type="radio"]');
-    cy.findOption('Y');
-    cy.realPress('Space');
-    cy.typeInIfDataExists('.year-input', '2019');
-    cy.tabToElement(
-      '[name="root_view:separationPayDetails_separationPayBranch"]',
-    );
-    cy.chooseSelectOptionByTyping('Army');
-
-    // 2. Can indicate that they never received separation pay
-    cy.tabToElement('[type="radio"]', false);
-    cy.findOption('N');
-
-    cy.tabToContinueFormSimulatedEnter();
-
-    // I. Veteran Information > J. Retirement Pay
-    // ==========================================
-    cy.url().should('include', veteranDetailsPages.retirementPay.path);
-    cy.injectAxeThenAxeCheck();
-
-    // 1. Can provide retirement pay details, if they exist
-    cy.tabToElement('[type="radio"]');
-    cy.findOption('Y');
-    cy.realPress('Space');
-    cy.tabToElement('#root_militaryRetiredPayBranch');
-    cy.chooseSelectOptionByTyping('Army');
-
-    // 2. Can indicate that they never received retirement pay
-    cy.tabToElement('[type="radio"]', false);
-    cy.findOption('N');
-
-    cy.tabToContinueFormSimulatedEnter();
-
-    // I. Veteran Information > K. Training Pay
-    // ========================================
-    cy.url().should('include', veteranDetailsPages.trainingPay.path);
-    cy.injectAxeThenAxeCheck();
-
-    cy.tabToElement('[type="radio"]');
-    cy.findOption('Y');
-    cy.realPress('Space');
-
-    cy.findOption('N');
-
-    cy.tabToContinueFormSimulatedEnter();
-
-    // II. Disabilities
-    cy.url().should('include', disabilitiesPages.addDisabilities.path);
-    cy.injectAxeThenAxeCheck();
-  });
-});
+// import path from 'path';
+// import { WIZARD_STATUS_COMPLETE } from 'platform/site-wide/wizard';
+// import formConfig from '../config/form';
+
+// import mockUser from './fixtures/mocks/user.json';
+// import mockFeatureToggles from './fixtures/mocks/feature-toggles.json';
+// import mockInProgress from './fixtures/mocks/in-progress-forms.json';
+// import mockLocations from './fixtures/mocks/separation-locations.json';
+// import mockPayment from './fixtures/mocks/payment-information.json';
+// import mockUpload from './fixtures/mocks/document-upload.json';
+// import mockSubmit from './fixtures/mocks/application-submit.json';
+
+// import { mockItf } from './cypress.helpers';
+
+// import {
+//   MOCK_SIPS_API,
+//   WIZARD_STATUS,
+//   FORM_STATUS_BDD,
+//   SHOW_8940_4192,
+// } from '../constants';
+
+// TODO: Revise and refactor in future iteration
+
```

Comment:
This test was updated to be skipped 11/28/2023 - it is slated to be re-worked in future product iterations, but will sometimes fail stability testing. For this reason, it is being removed temporarily.